### DISCUSSION TITLE
[Idea] `MaterialChoiceTableType` introduce `display_total_items` form `option`

### DISCUSSION THREAD

**Discussion by PululuK** [PrestaShop/PrestaShop#35551] [Ideas]
### The problem

`MaterialChoiceTableType` introduce `display_total_items` form `option`

### The solution

![Capture d’écran du 2024-03-04 11-09-23](https://github.com/PrestaShop/PrestaShop/assets/16455155/6056fab3-a425-468b-a104-b0d37a1b0db5)
![Capture d’écran du 2024-03-04 11-08-40](https://github.com/PrestaShop/PrestaShop/assets/16455155/4b7d6669-7f19-4305-9886-3da78b4a7a83)

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
Nice idea ! Don't hesitate to propose a PR ;)

**Reply by PululuK** [reply]
Hello @MatShir , done https://github.com/PrestaShop/PrestaShop/pull/35552
